support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -1,95 +1,63 @@
-  idWebAppUrl: IdentityUrl,
-  supportUrl: String,
+  val asyncAuthenticationService: AsyncAuthenticationService,
   cc: ControllerComponents,
   addToken: CSRFAddToken,
   checkToken: CSRFCheck,
   csrfConfig: CSRFConfig,
   stage: Stage
 )(implicit private val ec: ExecutionContext) {
 
-  import CustomActionBuilders._
-
-  val membersIdentityClientId = "members"
-  val subscriptionsClientId = "subscriptions"
-  val recurringIdentityClientId = "recurringContributions"
-
-  // Tells identity to send users back to the checkout immediately after sign-up.
-  private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
-  // Prevents the identity validation email sending users back to our checkout.
-  private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
-
-  private def idWebAppRegisterUrl(path: String, clientId: String): String =
-    (idWebAppUrl.value / "signin" ?
-      ("returnUrl" -> s"$supportUrl$path") &
-      idSkipConfirmation &
-      idSkipValidationReturn &
-      "clientId" -> clientId &
-      "componentEventParams" -> "componentType=identityauthentication&componentId=signin_redirect_for_supporters").toString
-
-  def onUnauthenticated(identityClientId: String): RequestHeader => Result = request => {
-    SeeOther(idWebAppRegisterUrl(request.uri, identityClientId))
-  }
-
-  private def maybeAuthenticated(onUnauthenticated: RequestHeader => Result): ActionBuilder[OptionalAuthRequest, AnyContent] =
-    new AsyncAuthenticatedBuilder(
-      asyncAuthenticationService.tryAuthenticateUser,
-      cc.parsers.defaultBodyParser,
-      onUnauthenticated
-    )
-
-  private def authenticated(onUnauthenticated: RequestHeader => Result): ActionBuilder[AuthRequest, AnyContent] =
-    new AsyncAuthenticatedBuilder(asyncAuthenticationService.authenticateUser, cc.parsers.defaultBodyParser, onUnauthenticated)
-
   val PrivateAction = new PrivateActionBuilder(addToken, checkToken, csrfConfig, cc.parsers.defaultBodyParser, cc.executionContext)
 
-  def authenticatedAction(identityClientId: String = membersIdentityClientId): ActionBuilder[AuthRequest, AnyContent] = {
-    PrivateAction andThen authenticated(onUnauthenticated(identityClientId))
-  }
-
-  def maybeAuthenticatedAction(identityClientId: String = membersIdentityClientId): ActionBuilder[OptionalAuthRequest, AnyContent] =
-    PrivateAction andThen maybeAuthenticated(onUnauthenticated(identityClientId))
-
-  def alarmOnFailure[A](action: Action[A]): EssentialAction =
-    (v1: RequestHeader) => action.apply(v1).map { result =>
-      if (result.header.status.toString.head != '2') {
-        SafeLogger.error(scrub"pushing alarm metric - non 2xx response ${result.toString()}")
+  val MaybeAuthenticatedAction: ActionBuilder[OptionalAuthRequest, AnyContent] =
+    PrivateAction andThen new AsyncAuthenticatedBuilder(asyncAuthenticationService.tryAuthenticateUser, cc.parsers.defaultBodyParser)
```

Comment:
the diff is not really lining up, but this top part of the file is mostly deleting authenticated action stuff that's unused.  Below this point I have combined the old `alarmOnFailure` action, plus the `LoggingCirceParser` from the CreateSubscriptionController.  I have also simplified it and made it follow the examples in the documentation in terms of syntax/usage https://www.playframework.com/documentation/2.8.x/ScalaActionsComposition